graphql/src/store/resolver.rs

```diff
@@ -141,6 +155,45 @@ impl StoreResolver {
             }
         }
     }
+
+    fn handle_meta(
+        &self,
+        prefetched_object: Option<q::Value>,
+        object_type: &ObjectOrInterface<'_>,
+    ) -> Result<(Option<q::Value>, Option<q::Value>), QueryExecutionError> {
+        // Pretend that the whole `_meta` field was loaded by prefetch. Eager
+        // loading this is ok until we add more information to this field
+        // that would force us to query the database; when that happens, we
+        // need to switch to loading on demand
+        if object_type.is_meta() {
+            let hash = self
+                .block_ptr
+                .and_then(|ptr| {
+                    if ptr.hash == web3::types::H256::zero() {
+                        None
```

Comment:
Worth a comment and a `See also` linked by uuid to the place where we set the hash to zero.

Reply:
Done